samples/tools/testbed/utils/download_humaneval.py

```diff
@@ -0,0 +1,67 @@
+#
+# Run this file to download the human_eval dataset, and create a corresponding testbed scenario:
+# (default: ../scenarios/human_eval_two_agents_gpt4.jsonl and ./scenarios/human_eval_two_agents_gpt35.jsonl)
+#
+
+import requests
+import gzip
+import io
+import json
+import os
+import base64
+
+
+script_path = os.path.realpath(__file__)
+script_name = os.path.basename(script_path)
+script_dir = os.path.dirname(script_path)
+
+# Directory where scenarios are stored
+scenarios_dir = os.path.realpath(os.path.join(script_dir, os.path.pardir, "scenarios"))
+print("Saving HumanEval scenarios to: " + scenarios_dir)
+
+
+# URL of the file to download
+url = "https://github.com/openai/human-eval/raw/master/data/HumanEval.jsonl.gz"
```

Comment:
Console downloading a file into a utility function

Reply:
I'll mark this as resolved. Please re-open this thread if this is a blocking issue.